pkg/http/http.go

```diff
@@ -144,6 +147,10 @@ func performRequest(req *http.Request, verifyTLS bool, params []queryParam) (int
 	requestErr := retry(5, 100*time.Millisecond, func() error {
 		resp, err := client.Do(req)
 		if err != nil {
+			if resp != nil {
+				defer response.Body.Close()
```

Comment:
`response` is still nil here, should this be `resp`?

Reply:
Great catch